content/hardware/08.edu/solution-and-kits/alvik/tutorials/01.getting-started/getting-started.md

```diff
@@ -90,7 +90,7 @@ The latest firmware of the STM32 can be found at [this link](https://github.com/
 
 At the back-right side of Alvik there is the main switch of the robot. When ON the robot will power up and it will execute the already loaded program.
 
-***While programming the terminal of the Arduino Lab for MicroPython will notify you if you forgot to switch on the robot. Keep the robot off while programming to avoid undesired movements and remember to turn it on when you are ready to execute your program***
+***Keep the robot off while programming to avoid undesired movements and remember to turn it on when you are ready to execute your program. A feature of the Arduino Lab for MicroPython terminal is that it will notify you if you forgot to switch on the robot to run the code. ***

```

Comment:
```suggestion
***Keep the robot off while programming to avoid undesired movements and remember to turn it on when you are ready to execute your program. A feature of the Arduino Lab for MicroPython terminal is that it will notify you if you forgot to switch on the robot to run the code.***

```